.github/workflows/continuous_integration.yaml

```diff
@@ -83,5 +83,5 @@ jobs:
       - uses: actions/checkout@v2
       - uses: ./.github/actions/poetry_deps
         with:
-          poetry-dir: performance_manager
+          poetry-dir: python_src
       - run: poetry run pytest
```

Comment:
this test should be changed to only run the tests that we need based on what has changed.

i have made [a ticket](https://app.asana.com/0/0/1204188000578032/f) to do so.

we can create an action that looks at what files have been changed and create bools for what tests we need to run.